storage/google/cloud/storage/blob.py

```diff
@@ -852,6 +857,31 @@ def rewrite(self, source, token=None, client=None):
 
         return api_response['rewriteToken'], rewritten, size
 
+    def update_storage_class(self, new_class, client=None):
```

Comment:
Ah. Blerg. We definitely need a tracking issue for that (kicking the can down the road is better than deleting it from our memory)